instruqt/delivering-as-an-appliance/02-customize-for-your-app/assignment.md

```diff
@@ -1,21 +1,205 @@
 ---
 slug: customize-for-your-app
-id: nk3jmszcvy6l
+id: 6003pttnmqok
 type: challenge
 title: Customizing the Appliance for Your Application
 teaser: Make the Kubernetes appliance experience your own
 notes:
 - type: text
-  contents: Customize the console to represent your application
+  contents: Customize the Admin Console to represent your application
 tabs:
-- title: Shell
+- id: ybi0oubv14zx
+  title: Release Editor
+  type: code
+  hostname: shell
+  path: /home/replicant
+- id: qgpmbdncfnse
+  title: Shell
   type: terminal
   hostname: shell
 difficulty: basic
 timelimit: 300
 ---
 
 The Kubernetes appliance installation is one of the first things
-your customer will experience about your application. Replicated
-provides a few options to customize the appliance installer. Let's
-use these to tailor it for the Slackernews application.
+your customer will experience about your application. The way we've implemented
+it so far uses a lot of defaults. We didn't try to install it yet, but if we
+had, we'd land on a page like this:
+
+![Appliance login page without branding](../assets/unbranded-admin-console.png)
+
+This definitely violates our brand standards!
+
+Don't worry, Replicated provides a few options to customize the appliance
+installer. The first customizations we're going to look at will make it clear
```

Comment:
"to customize the appliance installer" --> "customize the Admin Console" (maybe just to be very specific and match the title of the section?)

Reply:
I split the difference 😉